### PR TITLE
Changed package name to easy-path-expressions

### PR DESCRIPTION
**Pull request by mazilumt**
Changed the package name and supporting documentation to `easy-path-expressions`

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,12 +2,16 @@
...
 ## [1.1.0] 2021-04-13
 - Support for logical operators
 - Added operator validation
-- Fixed bugs in variable idenfication
\ No newline at end of file
+- Fixed bugs in variable identification
+
+## [1.0.0] 2021-03-22
+### Changed
+- Variable support for string variables
```
Comment (plynchnlm) [inline CHANGELOG.md:16]: Did you add these lines for version 1.0.0?

File: README.md [hunk 2]
```diff
@@ -18,12 +18,12 @@ npm test
 # Usage
 
 ```javascript
-import mathtofhirpath
-mathtofhirpath.fhirconvert("(a+b)^2", [a, b]); // returns ($a+$b).power(2)
-mathtofhirpath.fhirconvert("CEILING(a*b)", [a, b]); // returns ($a*$b).ceiling()
+import * as easyPathExpressions from 'easy-path-expressions';
+easyPathExpressions.fhirconvert("(a+b)^2", [a, b]); // returns ($a+$b).power(2)
```
Comment (plynchnlm) [inline README.md]: I was testing this earlier.  [a, b] should be ["a", "b"].

File: README.md [hunk 3]
```diff
@@ -1,15 +1,15 @@
-# Math-to-fhirpath
+# Easy Path Expressions
 
-Math-to-fhirpath is a syntax parser that converts basic mathematical expressions
-into FHIRpath notation. 
+Easy-path-expressions is a syntax parser that converts basic mathematical
+expressions into FHIRPath notation.
```
Comment (plynchnlm) [inline README.md]: Make FHIRPath be a link to FHIRPath.

File: package.json [hunk 4]
```diff
@@ -1,16 +1,12 @@
 {
-  "name": "js-to-fhirpath",
-  "version": "1.1.0",
+  "name": "easy-path-expressions",
+  "version": "2.0.0",
   "type": "module",
-  "description": "js to fhirpath syntax converter",
+  "description": "JS to FHIRPath syntax converter",
```
Comment (plynchnlm) [inline package.json]: In the README, you say it "converts basic mathematical expressions into FHIRPath", which I think is more accurate.

**Review by plynchnlm — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (mazilumt) [reply CHANGELOG.md:16]: I did not. I reversed the order of the changelog and fixed a typo.

File: README.md (hunk 2 above)
Comment (mazilumt) [reply README.md]: Good catch, updated.

File: package.json (hunk 4 above)
Comment (mazilumt) [reply package.json]: Updated.

File: README.md (hunk 3 above)
Comment (mazilumt) [reply README.md]: Updated.

**Review by plynchnlm — APPROVED** [review]

**Pull request merged**